app/views/user/cooking_sessions/new.html.erb

```diff
@@ -1,11 +1,12 @@
-<div class="container">
-  <h5>Let's schedule a cooking session!</h5>
+<div class="container mt-3">
+  <h4>You just added <strong><%= @meals.last.name %></strong> on your dish list</h4>
```

Comment:
This shouldn't show if a chef just clicks the link from their dashboard.
Can we add a comment to go back to this?